egs/wsj/s5/steps/nnet3/nnet3_train_lib.py

```diff
@@ -57,8 +57,12 @@ def CheckIfCudaCompiled():
 
 
 class KaldiCommandException(Exception):
-    def __init__(self, command, err):
+    def __init__(self, command, err, stdout=None, code=None):
         Exception.__init__(self,"There was an error while running the command {0}\n".format(command)+"-"*10+"\n"+err)
+        self.stderr = err;
+        self.stdout = stdout;
+        self.command = command;
+        self.code = code
```

Comment:
The way I want to change the RunKaldiCommand stuff is to just join the subprocess stdout/stderr to the parent process's stdout/stderr.   [stdout should not matter.]
That way, any errors or warnings generated by queue.pl  will be printed immediately.